redis.go

```diff
@@ -289,3 +258,116 @@ func NewRedisClusterStore(options *RedisClusterOptions, expiration time.Duration
 		expiration: expiration,
 	}, nil
 }
+
+// Pipeline uses pipeline as a Redis client to execute multiple calls at once
+func (r *RedisStore) Pipeline(f func(r *RedisStore) error) error {
+	pipe := r.client.Pipeline()
+
+	redisPipeline := RedisPipeline{
+		pipeline: pipe,
+	}
+
+	store := &RedisStore{
+		client:     redisPipeline,
```

Comment:
I have the following error : 
```
./redis.go:271: cannot use redisPipeline (type RedisPipeline) as type RedisClient in field value:
	RedisPipeline does not implement RedisClient (Close method has pointer receiver)````

@gillesfabio @thoas  can you explain me why? :/